### DISCUSSION TITLE
We got top Rust tending today

### DISCUSSION THREAD

**Discussion by linrium** [pintariching/rustle#26] [General]
A good start :) 

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/14315537/184527212-1b9b19d6-7d75-4592-b1e8-dddd231650e1.png">

**Comment by pintariching** [comment]
Holy crap, this was supposed to be a small side project for fun, whoops :sweat_smile:. Thank you everybody for working on this!!

**Comment by azzamsa** [comment]
![Screenshot_20220816-211420_GitHub.jpg](https://user-images.githubusercontent.com/17734314/184901861-22a55fab-5dcf-461c-9e34-44d2bba6fbb5.jpg)

Wow. That much stargazers in a single day!

**Reply by pintariching** [reply]
Wow. I guess people like the idea? :sweat_smile:
